harc_game_web/static/dist/rpgui.css

```diff
@@ -570,7 +570,8 @@ ul.rpgui-dropdown-imp {
 /* note! this class rule affect both the dropdown header and the list elements! */
 .rpgui-list-imp {
   /* font */
-  text-shadow: -2px 0 black, 0 2px black, 2px 0 black, 0 -2px black;
+  /* text-shadow do poprawki - w tabelkach pokazywal bardzo rozmyte teksty, przynajmniej na Safari */
```

Comment:
w moim PR częściowo ogarniam to, napisz proszę które dokładnie elementy miały taki problem

Reply:
W tabelkach na stronie /tasts/